time/src/main/proto/spine/time/time_change.proto

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2018, TeamDev. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.time;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_multiple_files = true;
+option java_outer_classname = "TimeChangeProto";
+option java_package = "io.spine.time";
+
+import "spine/time/time.proto";
+
+//
+// This file provides messages that define changes of date-time fields of the types defined 
+// by the Spine Time library.
+//
+// For change types for non-date/time values and for the `TimestampChange` message definition 
+// please see `spine/change/change.proto`.
+//
+// Messages defined in this file follow the conventions defined by the `change.proto`.
+//
+
+//
+// Changes of date/time values.
+//--------------------------------
+
+// Definition of a change of `LocalDate` fields.
+message LocalDateChange {
+
+    // The value of fields that are changing.
```

Comment:
The message only covers one field, so it should be singular.

Also seems to me like it’s not a value that is changed, but a field, making it a passive voice, so it should be `changed`?

Reply:
The change objects are for describing a coming change (when they come in a command), or already happened change (when the are in an event). So, we need a language that looks OK for both cases.